pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -91,6 +91,7 @@ type BackupStorageConfig struct {
 	AzureStorageContainer string                `json:"azureStorageContainer,omitempty"`
 	AzureResourceGroup    string                `json:"azureResourceGroup,omitempty"`
 	GcpBucket             string                `json:"gcpBucket,omitempty"`
+	AwsCustomCABundle     []byte                `json:"awsCustomCABundle,omitempty"`
```

Comment:
I like having the caBundle directly on the MigStorage because it simplifies the process of providing this information. 

I am wondering if this will cause a burdensome CLI editing experience for MigStorage, since caBundles can be very large and can contain newlines, and putting this directly on resource means it won't be b64 encoded and will keep those newline characters intact.

Something to keep in mind but not recommending we change the current approach.

Reply:
> and putting this directly on resource means it won't be b64 encoded and will keep those newline characters intact.

Sorry, I'm not following-- why does having this on the resource inherently mean it won't be b64 encoded?

Reply:
You *can* have b64 encoded data directly on a resource, but *all* data isn't shown in b64 *by default* when you look at `oc edit migstorage foo` like it would be if you were looking at a secret with `oc edit secret foo`.

Now that I think of it, configMaps also aren't b64 encoded, and I've seen caBundles dumped in configMaps without issue before so don't think my original concern was valid. Marking this as resolved.

Reply:
Gotcha, thanks for the explanation!